File: src/API/src/Modules/Lookups/Lookups.Application/ViewModels/LookupListItemViewModel.cs
namespace Lookups.Application.ViewModels
{
    public class LookupListItemViewModel
    {
        public Guid Id { get; set; }
        public int LookupId { get; set; }
        public string LookupName { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public int Order { get; set; }
        public string? DialingCode { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}

File: src/ClientApp/src/app/pages/lookups/view-models/lookup-list-item.view-model.ts
export interface LookupListItemViewModel {
    id: string;
    lookupId: number;
    lookupName: string;
    name: string;
    description?: string | null;
    order: number;
    dialingCode?: string | null;
    isActive: boolean;
    createdOn: string;
}
